vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -241,6 +241,17 @@ func NewTooManyRequests(message string, retryAfterSeconds int) *StatusError {
 	}}
 }
 
+// NewRequestEntityTooLargeError returns an error indicating that the request
+// entity was too large.
+func NewRequestEntityTooLargeError(message string) *StatusError {
```

Comment:
Added manually as it was missing - had been added before #74000, https://github.com/kubernetes/kubernetes/commit/b971b12d3cfa8887aa87957fbba92b90e3e99dc6

Reply:
just ignore my previous comment, this method was added in https://github.com/openshift/origin/pull/22267/commits/f22e9205fb8e64e37e36be5c96b7eb13f3d55aad